client/extensions/wp-super-cache/caching.jsx

```diff
@@ -45,6 +50,19 @@ const Caching = ( {
 			</SectionHeader>
 			<Card>
 				<form>
+					{ htaccess_ro && htaccess_ro.message &&
```

Comment:
Might be worth declaring an extra `const` (using lodash `get`) to use here and for the `text` prop below.